.42AI/init.sh

```diff
@@ -66,27 +66,32 @@ done
 ############################################################
 # .envrc                                                   #
 ############################################################
-echo -e $COLOR_YELLOW "INIT:" $COLOR_RESET "Checking if .envrc exist"
-if test -f "$PATH_ENVRC"; then
-    echo -e $COLOR_GREEN "$PATH_ENVRC exists." $COLOR_RESET
+echo -e $COLOR_YELLOW "INIT:" $COLOR_RESET "Checking if .twitter_keys.yaml exist"
+if test -f "$PATH_TWITTER_KEYS"; then
+    echo -e $COLOR_GREEN "$PATH_TWITTER_KEYS exists." $COLOR_RESET
 else
-    echo -e $COLOR_RED "ERROR: .envrc do not exist at location $PATH_ENVRC" $COLOR_RESET
+    echo -e $COLOR_RED "ERROR: .twitter_keys.yaml does not exist at location $PATH_TWITTER_KEYS" $COLOR_RESET
 	echo "It can be created this way:"
-	echo "$> echo \"source venv/bin/activate\" > .envrc"
-	exit 1
+ 	echo "$> echo \"<name_of_key:\n
```

Comment:
Should it be `<name_of_key:` or `<name_of_key>:` ?

Reply:
yes!